### ISSUE TITLE
Install failure on OSX with homebrew

### ISSUE THREAD

**Issue by robinbowes** [koalaman/shellcheck#348]
I have shellcheck 0.3.5 installed with homebrew on OSX 10.10.3.

When I try to update, I get this:
```
 robin at mbp in ~
  brew upgrade
==> Upgrading 1 outdated package, with result:
shellcheck 0.3.6
==> Upgrading shellcheck
==> Downloading https://github.com/koalaman/shellcheck/archive/v0.3.6.tar.gz
######################################################################## 100.0%
==> cabal sandbox init
==> cabal update
==> cabal install --jobs=8 --only-dependencies
==> cabal install --jobs=8 --prefix=/opt/boxen/homebrew/Cellar/shellcheck/0.3.6
               (nub . sortNotes . excludeNotes
                $ p ++ notesForContext context ++ [makeErrorFor err]) }
cabal: Error: some packages failed to install:
ShellCheck-0.3.6 failed during the building phase. The exception was:
ExitFailure 1
```

Here's the brew gist-logs output: https://gist.github.com/anonymous/7faf9bcc8fc1acd08581

**Comment by koalaman** [comment]
This is an issue in later versions of GHC, and it was fixed in 47b971c5 (right after the 0.3.6 release).

0.3.7 may be pushed out soon just to get this GHC fix plus an OS X regex fix out.

**Comment by robinbowes** [comment]
Good to know - I'll hold out for the 0.3.7 release.

Thanks,

R.
